app/ide-desktop/lib/dashboard/src/layouts/TabBar.tsx

```diff
@@ -218,6 +218,8 @@ export function Tab(props: InternalTabProps) {
       )}
     >
       <ariaComponents.Button
+        // Change this `button` into an `a` to avoid the DOM nesting error.
```

Comment:
This is not acceptable by the aria guidelines. We can't put anything interactive inside a link.

Reply:
I think the best solution is to use  https://react-spectrum.adobe.com/react-aria/Tabs.html